impl/src/metadata.rs

```diff
@@ -211,254 +81,11 @@ impl MetadataFetcher for CargoSubcommandMetadataFetcher {
       dir
     };
 
-    // Shell out to cargo
-    let exec_output = Command::new(&self.cargo_bin_path)
+    MetadataCommand::new()
+      .cargo_path(&self.cargo_bin_path)
```

Comment:
Wow I'm impressed that this just drops in here.

(no action item)

Reply:
\o/